### ISSUE TITLE
checkpoint_epoch_36_nus_mamba

### ISSUE THREAD

**Issue by Damoooon** [happinesslz/LION#14]
I tried to use checkpoint_epoch_36_nus_mamba.pth to make predictions on the nuscenes point cloud data and found that the score for each box was very low (almost no accurate boxes)
![image](https://github.com/user-attachments/assets/ffaa00b6-bcc5-474e-bc1b-2852d0ff9d01)

**Comment by AlmoonYsl** [comment]
@Damoooon 
Hi, can you get right performance (72.1 NDS, 68.0 mAP) by evaluating LION on the nuScenes validation set?

**Comment by Damoooon** [comment]
Hi, The result of eval is normal, but the strange thing is that for the same frame of point cloud data, the result of pred_dicts obtained by running test.py with the same model is very different from the result of pred_dicts obtained by using demo.py. Have you tried to visualize 3D prediction frames using demo.py

**Comment by Damoooon** [comment]
> @Damoooon Hi, can you get right performance (72.1 NDS, 68.0 mAP) by evaluating LION on the nuScenes validation set?

Hi, The result of eval is normal, but the strange thing is that for the same frame of point cloud data, the result of pred_dicts obtained by running test.py with the same model is very different from the result of pred_dicts obtained by using demo.py. Have you tried to visualize 3D prediction frames using demo.py

**Comment by AlmoonYsl** [comment]
@Damoooon 
Hi, you should use the ```data_processor``` of nuScenes dataset to process input data for inference. You can refer to the ```test.py``` for creating the dataloader and testing LION.

**Comment by Damoooon** [comment]
> @Damoooon Hi, you should use the `data_processor` of nuScenes dataset to process input data for inference. You can refer to the `test.py` for creating the dataloader and testing LION.

ok, But if DemoDataset doesn't work then how do I inference to my own data?

**Comment by AlmoonYsl** [comment]
@Damoooon 
Hi, if you want to use your data, you should retrain LION with your data.

**Comment by Damoooon** [comment]
I have solved the problem, checkpoint_epoch_36_nus_mamba.pth uses multiframe for training, but the dataloader in demo.py is singleframe, which maybe caused the data domain difference between the training data and the demo data, if use the The singleframe trained model can solve the problem of poor prediction result of demo.py, for example, use argov2_mamba.pth, or change the dataloader in demo.py to multiframe
